### PR TITLE
[MLP-115] Create Xamarin seed project

### PR DESCRIPTION
**Pull request by mvazquez-ms**
Hi team,

I've created the initial project to work on a Xamarin seed, targeting mainly Android and iOS. The objective is to have a seed project with an MVVM architecture, IoC, some local storage mechanism and a couple of useful classes that can be easily re-used on multiple projects.

The only highlight of this PR is the usage of _.NET Standard 1.6_. as a replacement to profile-based PCLs (ccing @AlphaGit and @mravinale so you are aware of this). Configuration might be a bit tricky, but seems to be working fine and I think it is the way to go.

### CODE REVIEW

**Review by matiasbeckerle — APPROVED** [review]
Looking good!

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+#*.doc   diff=astextplain
+#*.DOC   diff=astextplain
+#*.docx  diff=astextplain
+#*.DOCX  diff=astextplain
+#*.dot   diff=astextplain
+#*.DOT   diff=astextplain
+#*.pdf   diff=astextplain
+#*.PDF   diff=astextplain
+#*.rtf   diff=astextplain
+#*.RTF   diff=astextplain
```
Comment (AlphaGit) [inline .gitattributes]: Should we uncomment all of these settings to apply them? All of them are commented but they all seem like a good idea.

File: Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Properties/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="MvvmSeed.Android.MvvmSeed.Android" android:versionCode="1" android:versionName="1.0">
+	<uses-sdk />
```
Comment (AlphaGit) [inline Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Properties/AndroidManifest.xml]: Is this element used at all if the API versions are not indicated?

**Review by AlphaGit — COMMENTED** [review]
Looks good! Also, should we document what is required for .NET 1.6?

File: .gitattributes (hunk 1 above)
Comment (mvazquez-ms) [reply .gitattributes]: Sure, why not? I'll uncomment them.

File: Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Properties/AndroidManifest.xml (hunk 2 above)
Comment (mvazquez-ms) [reply Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Properties/AndroidManifest.xml]: Well, it's tricky.. I remember Android complaining about Manifest not having that tag but Xamarin doesn't rely on it.. I'll check and let you know.

**Review by mmaldini33 — APPROVED** [review]
Approved!

File: .gitattributes (hunk 1 above)
Comment (mvazquez-ms) [reply .gitattributes]: Done

File: Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Properties/AndroidManifest.xml (hunk 2 above)
Comment (mvazquez-ms) [reply Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Properties/AndroidManifest.xml]: Changed to specify minSdk version as 16 (4.1 and above)

**Review by AlphaGit — APPROVED** [review]

File: Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Properties/AndroidManifest.xml (hunk 2 above)
Comment (AlphaGit) [reply Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Android/Properties/AndroidManifest.xml]: Thanks!

File: .gitattributes (hunk 1 above)
Comment (AlphaGit) [reply .gitattributes]: Thank you!

**Comment by AlphaGit** [comment]
[SPAM]

image

Lol, I didn't even know this was possible.

**Pull request merged**